TreeTracker/UI/Views/Authentication/Selfie/SelfieViewController.swift

```diff
@@ -43,14 +48,22 @@ class SelfieViewController: UIViewController, AlertPresenting {
     override func viewDidLoad() {
         super.viewDidLoad()
         viewModel?.image = nil
+        self.fromLibraryButton.isHidden = true
```

Comment:
Maybe this logic could live in the `fromLibraryButton` `didSet` rather than here. It will help keep the setup code of UI components in one place.

Reply:
You're right. It's fixed 👍